client/packages/inventory/src/Stocktake/DetailView/DetailView.tsx

```diff
@@ -17,31 +17,29 @@ import { Footer } from './Footer';
 import { AppBarButtons } from './AppBarButtons';
 import { SidePanel } from './SidePanel';
 import { StocktakeSummaryItem } from '../../types';
-
 import { StocktakeLineEdit } from './modal/StocktakeLineEdit';
 import { ContentArea } from './ContentArea';
 import { AppRoute } from '@openmsupply-client/config';
-import { StocktakeLineFragment, useStocktake } from '../api';
+import { StocktakeFragment, StocktakeLineFragment, useStocktake } from '../api';
 import { StocktakeLineErrorProvider } from '../context';
+import { isStocktakeDisabled } from '../../utils';
 
-export const DetailView: FC = () => {
-  const isDisabled = useStocktake.utils.isDisabled();
-  const { isOpen, entity, onOpen, onClose, mode } =
-    useEditModal<ItemRowFragment>();
-  const { data, isLoading } = useStocktake.document.get();
-  const navigate = useNavigate();
-  const t = useTranslation('inventory');
-  const { HighlightStyles } = useRowHighlight();
-
+const StocktakeTabs = ({
```

Comment:
no, it didn't help much. I've kept it in anyway. 
there was re-rendering due to the TableStore being at the top level, so when that updated all the components had to update. this helps a little